openstudiocore/src/model/test/SiteGroundTemperatureDeep_GTest.cpp

```diff
@@ -0,0 +1,191 @@
+/**********************************************************************
+*  Copyright (c) 2008-2016, Alliance for Sustainable Energy.
+*  All rights reserved.
+*
+*  This library is free software; you can redistribute it and/or
+*  modify it under the terms of the GNU Lesser General Public
+*  License as published by the Free Software Foundation; either
+*  version 2.1 of the License, or (at your option) any later version.
+*
+*  This library is distributed in the hope that it will be useful,
+*  but WITHOUT ANY WARRANTY; without even the implied warranty of
+*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+*  Lesser General Public License for more details.
+*
+*  You should have received a copy of the GNU Lesser General Public
+*  License along with this library; if not, write to the Free Software
+*  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+**********************************************************************/
+
+#include <vector>
+#include <gtest/gtest.h>
+#include "ModelFixture.hpp"
+#include "../SiteGroundTemperatureDeep.hpp"
+#include "../SiteGroundTemperatureDeep_Impl.hpp"
+#include "../Model.hpp"
+
+using namespace openstudio::model;
+
+TEST_F(ModelFixture, SiteGroundTemperatureDeep_SiteGroundTemperatureDeep)
+{
+  ::testing::FLAGS_gtest_death_test_style = "threadsafe";
+
+  ASSERT_EXIT (
+    {
+      // create a model to use
+      Model m;
+
+      // create a SiteGroundTemperatureDeep object
+      SiteGroundTemperatureDeep ground_temp(m);
+      exit(0);
+    },
+    ::testing::ExitedWithCode(0),
+    ""
+  );
+
+  // create a model to use
+  Model m;
+
+  // create a SiteGroundTemperatureDeep object
+  SiteGroundTemperatureDeep ground_temp(m);
+
+  // Check to make sure all the fields are defaulted as expected
+  ASSERT_TRUE(ground_temp.isJanuaryDeepGroundTemperatureDefaulted());
```

Comment:
Minor note, you can use EXPECT_TRUE here instead, ASSERT_TRUE stops the test on failure, EXPECT_TRUE continues on